.pre-commit-config.yaml

```diff
@@ -46,7 +46,7 @@ repos:
         args:
           - --min-py3-version
           - "3.4"
-  - repo: https://gitlab.com/pycqa/flake8
+  - repo: https://github.com/PyCQA/flake8
```

Comment:
shoulda just waited, pre-commit.ci is going to do this for you

Reply:
Thanks! I did not know that!

I ran had to run `all-repos` anyway for my other projects, so this was more or less a bycatch.